### ISSUE TITLE
[Feature] More console history

### ISSUE THREAD

**Issue by Bloodlex** [MCSManager/MCSManager#1369]
### Description

Is it possible to browse more console history?

### Reason

It would be easier to check what happened on a server.

**Comment by Bloodlex** [comment]
> check log files.

Yes I know, I can check the logs, but isn't it technically possible to allow to browse more history inside terminal window?

If not, maybe some way to browse the old logs right from the panel directly?

**Comment by huangsijun17** [comment]
> > check log files.
> 
> Yes I know, I can check the logs, but isn't it technically possible to allow to browse more history inside terminal window?
> 
> If not, maybe some way to browse the old logs right from the panel directly?

The current terminal has shown thousands of lines of historical logs. There are already a lot for the command terminal.
If you need to view a large number of historical logs, I suggest to check the log file in the file management.
